plugins/faustwp/includes/replacement/callbacks.php

```diff
@@ -233,8 +233,22 @@ function term_link( $term_link ) {
  * XXX: Please remove this once this issue is resolved: https://github.com/WordPress/gutenberg/issues/13998
  */
 function enqueue_preview_scripts() {
-	wp_enqueue_script( 'faustwp-gutenberg-filters', plugins_url( '/previewlinks.js', __FILE__ ), array( 'jquery' ), plugin_version(), true );
-	wp_localize_script( 'faustwp-gutenberg-filters', '_faustwp_preview_link', array( '_preview_link' => get_preview_post_link() ) );
+	wp_enqueue_script(
+		'faustwp-gutenberg-filters',
+		plugins_url( '/previewlinks.js', __FILE__ ),
+		array(),
+		plugin_version(),
+		true
+	);
+
+	wp_localize_script(
+		'faustwp-gutenberg-filters',
+		'_faustwp_preview_link',
+		array(
+			'_preview_link' => get_preview_post_link(),
+			'_wp_version'   => get_bloginfo( 'version' ),
```

Comment:
Adding the WordPress version to better handle any breaking changes until https://github.com/WordPress/gutenberg/issues/13998 is resolved.